README.md

```diff
@@ -175,6 +175,17 @@ brew install diffsitter
 @samhh has packaged diffsitter for arch on the AUR. Use your favorite AUR
 helper to install [`diffsitter-bin`](https://aur.archlinux.org/packages/diffsitter-bin/).
 
+### Alpine Linux
+
+Install package [diffsitter](https://pkgs.alpinelinux.org/packages?name=diffsitter) from the Alpine Linux’s repositories (on v3.16+ or Edge):
```

Comment:
Small typo: change this to "from the Alpine Linux repositories"

Reply:
Okay, fixed.